common/app/services/ConfigAgentTrait.scala

```diff
@@ -54,12 +54,12 @@ object ConfigAgent extends GuLogging {
     configAgent.alter(Option(config))
   }
 
-  def refreshAndReturn(implicit ec: ExecutionContext): Future[Option[ConfigJson]] =
+  def refreshAndReturn(implicit ec: ExecutionContext): Future[Unit] =
```

Comment:
Would this function name still make sense if it's no longer returning anything?

Reply:
I'll merge as-is but very happy to open a separate PR to rename it if you'd prefer :)